BikeData/src/main/scala/by/itechart/internship/FileWriter.scala

```diff
@@ -0,0 +1,29 @@
+package by.itechart.internship
+
+import java.io.{BufferedWriter, FileWriter}
+
+import au.com.bytecode.opencsv.CSVWriter
+
+import scala.collection.JavaConverters._
+import scala.util.{Failure, Try}
+
+object FileWriter {
+  type BikeInfo = String
```

Comment:
I see BikeInfo  defined multiply times - it'd expect types to be declared once or have different names